### ISSUE TITLE
Importing LastPass Required Notepad++

### ISSUE THREAD

**Issue by compumatter** [nextcloud/passman#394]
I have over 800 credentials exported from Lastpass and pasted into Windows Notepad then saved as a csv file being careful to save using the Unicode setting.

I used the import feature within your plugin (thank you).

However, it failed without errors.  It appeared to begin some processes but quickly failed without warning or feedback.  I had the same problem in Firefox as in Chrome.

On a separate attempt I did everything the same but used Notepad++ instead of Notepad and the import worked perfectly.  

Wanted to pass this along as it appears that it is a bug even in Unicode if the write software is not used.  Strange but true.

Jay
CompuMatter

**Comment by animalillo** [comment]
Thank you for reporting it, the could make for a nice wiki page, as lastpass tends to cause import issues due to their awful export process.

**Comment by compumatter** [comment]
I agree.  If you are able to provide edit access for me I am happy to add this to your wiki.

**Comment by brantje** [comment]
```
Wikis 
GitHub Wikis is a simple way to let others contribute content. Any GitHub user can create and edit pages to use for documentation, examples, support, or anything you wish.
```

I think you _should_ be able to edit the wiki.

**Comment by compumatter** [comment]
No ability for me to edit the wiki.  I think it's a project owner thing.

**Comment by brantje** [comment]
Added you to the project, poke me when done =)

**Comment by lklimek** [comment]
Two things I noticed today, when working on Linux:

1. The issue is that lastpass export had some "binary" characters on it, like several characters of hex value  0x10 ; I had to replace them with spaces
2. My export used unix line ending, I used tool "unix2dos" to convert it into windows-styled line endings

After doing both, I have successfully run the import.

**Comment by timo12357** [comment]
Ubuntu user here:
I exported the passwords from Lastpass, got a webpage. 
Did ctrl-a ctrl-c to copy the passwords
Opened gedit
Pasted passwords to geditwith ctrl-v 
Deleted empty lines in beginning
Deleted 0x10 characters, did not replace with anything.
Saved file
Used unix2dos to convert to right format
Import success.

Prior to this I attempted quite a few different variations of the above until it worked.
